live-editing/configs/app-dv-configs/DVHierarchicalGridConfig.ts

```diff
@@ -28,6 +28,21 @@ export class DVHierarchicalGridConfigGenerator implements IConfigGenerator {
             component: 'HGridColumnResizingSampleComponent'
         }));
 
```

Comment:
you probably wanted to modify live-editing/configs/HierarchicalGridConfigGenerator.ts

Reply:
yes, moved now